index.html

```diff
@@ -139,153 +113,124 @@
                     <div>
-                        <h1>hackSocIndore</h1>
+                        <h5 class="text-uppercase">Culture</h5>
+                        <div>
+                            <h1>About</h1>
+                        </div>
+                        <div>
+                            <h1>hackSocIndore</h1>
+                        </div>
                     </div>
                 </div>
-            </div>
-            <div class="row col-sm-12 col-md-9 align-items-center mr-0 pr-0">
-                <div class="col-sm-8 center">
-                    <p>"hackSoc? as in hacking computers?" <br>Although we have members who can do that, we do NOT collectively do that here.</p>
-
-                    <p> hackSoc is <b>a community of developers that hosts workshops, tech meetups, Hack Days regularly, and enables project collaboration among its members!</b>
-                        <br/><br/>We believe in "learning by doing", so bring along your laptops to the workshops to start coding.
-                        Coding is awesome and we love to help you get started. Just come along to any of the free
-                        sessions and let's get set code!
-                    </p>
-                </div>
-                <div class="col-sm-4 text-right mr-0 pr-0 ml-0 pl-0" style="width: 100%; overflow-x: hidden;">
-                    <img height="448px" src="img/about.jpg" alt="Image">
+                <div class="for-mobile row col-sm-12 col-md-9 align-items-center mr-0 pr-0">
+                    <div class="fs-3 col-sm-8 center">
+                        <p>"hackSoc? as in hacking computers?" <br />Although we have members who can do that, we do NOT collectively do that here.</p>
+
+                        <p>hackSoc is <b>a community of developers that hosts workshops, tech meetups, Hack Days regularly, and enables project collaboration among its members!</b> <br /><br />We believe in "learning by doing", so bring along your laptops to the workshops to start coding. Coding is awesome and we love to help you get started. Just come along to any of the free sessions and let's get set code!</p>
+                    </div>
+                    <div class="col-sm-4 text-right mr-0 pr-0 ml-0 pl-0" style="width: 100%; overflow-x: hidden;">
+                        <img height="448px" src="img/about.jpg" alt="Image" />
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
-    <!-- End Culture -->
-
-    <!--========== Contact Us ==========-->
-
-    <div class="cu" >
-        <div class="text-group text-center">
-            <h3 class="h2">Contact Us</h3>
-            <p class="para_cu"  >For any queries.</p>
-        </div>
-
-        <div style="text-align: center;">
-            <a id="cu_a"  href="mailto:hacksocindore@gmail.com" target="_top"  class=" btn btn-secondary btn-lg" >SEND US AN EMAIL!</a>
-        </div>
-    </div>
-
-    <!--========== Contact Us  End==========-->
```

Comment:
Why is Contact Us section and its classes are deleted?

Reply:
will fix it in a while